### PR TITLE
Replace empty array key by none when nobody is assigned to a JiraTicket

### LINKED ISSUE #44: Display "None" when some people is not assigned to a JiraTicket
## Current behavior

Currently, while running the `examples/using-cli-channel`, when nobody is assigned to a JiraTicket then it will render nothing.

## Acceptance Criteria

We should render "None", instead of nothing.

### PR DESCRIPTION
**Pull request by JesusValeraDev**
# Description

I am not sure which name is better for the `replaceEmptyKeyArray()` function.
If `replaceEmptyKeyArray`, `replaceNullByNoneIfNobodyAssignedToTicket` or any other construction.
Also, does it worth to put a default value or just put directly `None` as array key?

What do you think @Chemaclass?

Issue: #44 

# Changes

- [x] Add `\Chemaclass\ScrumMaster\Channel\ChannelResult::replaceKey()`.
- [x] Update `\Chemaclass\ScrumMasterTests\Unit\Channel\ChannelResultTest::ticketsAssignedToPeople()`.

# Checklist

- [x] Tests are passing
- [x] Mutants are defeated
- [x] Code coverage is not decreased

### CODE REVIEW

File: src/ScrumMaster/Channel/ChannelResult.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public function ticketsAssignedToPeople(): array
             }
         }
 
-        return $tickets;
+        return $this->replaceEmptyKeyArray($tickets);
```
Comment (Chemaclass) [inline src/ScrumMaster/Channel/ChannelResult.php]: What about `replaceKey($tickets, $from = '', $to = 'None');`?

**Review by Chemaclass — CHANGES_REQUESTED** [review]
I like your solution. Anyway, I wrote you a suggestion to improve it :)

File: src/ScrumMaster/Channel/ChannelResult.php (hunk 1 above)
Comment (JesusValeraDev) [reply src/ScrumMaster/Channel/ChannelResult.php]: Sure!

File: src/ScrumMaster/Channel/ChannelResult.php [hunk 2]
```diff
@@ -95,4 +95,12 @@ private function peopleAssigned(): array
 
         return $people;
     }
+
+    private function replaceKey(array $tickets, string $from = '', string $to = 'None'): array
```
Comment (Chemaclass) [inline src/ScrumMaster/Channel/ChannelResult.php]: I wouldn't use a default value for `$from` neither for `$to`. I rather prefer to force these values explicit where this function is used.
```php
return $this->replaceKey($tickets, $from = '', $to = 'None');
```

**Review by Chemaclass — CHANGES_REQUESTED** [review]

**Review by Chemaclass — APPROVED** [review]
Perfect! 💅 🉑 🥇 👾

**Pull request merged**